lib/question.js

```diff
@@ -583,6 +598,18 @@ module.exports = {
    * @param {function} callback - A callback(err, submission_id) function.
    */
   saveSubmission(submission, variant, question, course, callback) {
+    // This is (hopefully) temporary code to debug an issue where chunk servers
+    // get stuck inside of a transaction. We can possibly remove it once we
+    // identify and fix the issue.
+    const timeoutId = setTimeout(() => {
+      logger.error('saveSubmission transaction was not closed within 60 seconds', {
+        variant_id: variant.id,
+        question_id: question.id,
+        course_id: course,
```

Comment:
Not that it really matters, but I think `course` is an object here? (and similarly in later changes)
```suggestion
        course_id: course.id,
```

Reply:
Good catch! Done in 3d0021d.